microsoft-edge/webview2/index.md

```diff
@@ -88,6 +88,9 @@ WebView2 apps can run on the following versions of Windows:
 *  Windows Server 2019
 *  Windows Server 2016
 
+WebView2 is supported in the following devices:
```

Comment:
```suggestion
WebView2 is supported on the following devices:
```

Reply:
I like the second option: WebView2 is also supported on the following devices:

Reply:
@nishitha-burman, I can make such changes locally in VS Code and then push commits to your PR, soon.

Reply:
WebView2 is also supported on the following devices: